lld/MachO/Driver.cpp

```diff
@@ -90,6 +90,9 @@ static std::optional<StringRef> findLibrary(StringRef name) {
     return entry->second;
 
   auto doFind = [&] {
+    // Special case for Csu support files.
```

Comment:
nit: would be helpful to either expand what Csu are ... (not a very common thing - I had  to look it up :) ) 

Thanks!

Reply:
Is the new comment ok?